facia-json/src/main/scala/com/gu/facia/client/models/Config.scala

```diff
@@ -10,6 +10,37 @@ case class Backfill(
   query: String
 )
 
+sealed trait Metadata
+
+case object Canonical extends Metadata
+
+case object Special extends Metadata
+
+case object Breaking extends Metadata
+
+object Metadata {
+
+  val tags = List("Canonical", "Breaking", "Special")
```

Comment:
Maybe it would be more useful as a mapping to the type; `Map[String, Metadata]`. You could then use it in the `reads` for example.